bootstrap/pkg/kfapp/gcp/gcp.go

```diff
@@ -614,66 +807,22 @@ func (gcp *Gcp) generateDMConfigs() error {
 	}
 	from := filepath.Join(sourceDir, "iam_bindings_template.yaml")
 	to := filepath.Join(gcpConfigDir, "iam_bindings.yaml")
-	iamBindings := map[string]string{
-		"from": from,
-		"to":   to,
-	}
-	iamBindingsErr := gcp.copyFile(iamBindings["from"], iamBindings["to"])
-	if iamBindingsErr != nil {
-		return fmt.Errorf("could not copy iam_bindings Error %v", iamBindingsErr)
+	if err := gcp.writeIamBindingsFile(from, to); err != nil {
+		return err
 	}
-	iamBindingsData, iamBindingsDataErr := ioutil.ReadFile(to) // just pass the file name
-	if iamBindingsDataErr != nil {
-		return fmt.Errorf("could not read %v Error %v", to, iamBindingsDataErr)
+
+	from = filepath.Join(sourceDir, CONFIG_FILE)
+	to = filepath.Join(gcpConfigDir, CONFIG_FILE)
+	if err := gcp.writeClusterConfig(from, to); err != nil {
```

Comment:
If we can't have a better function name, add a comment here that this is to replace the placeholder

Reply:
added.